project_stage_state/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import project_task
```

Comment:
License notice is needed in `__init__` as well

Reply:
@alexey-pelykh , Could you please check it now?

Reply:
Only license, w/o copyright